### PR TITLE
Various updates for SciPy 2023

### PR DESCRIPTION
**Pull request by maximlt**

### CODE REVIEW

File: examples/tutorial/02_Plotting.ipynb [hunk 1]
```diff
@@ -665,9 +665,57 @@
...
+    "Once you have found a representation of the data that you want to capture, you can record the configuration in a few ways:\n",
+    "- call `hvexplorer.settings()` to obtain a configuration dictionary (let's call it `settings`) that you can then unpack to `small_df.hvplot(**setting)` to recreate the same exact plot you see in the explorer\n",
+    "- call `hvexplorer.plot_code('small_df')` to print a string that you can copy and paste in another cell and that will again recreate the same exact plot you see in the explorer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Don't feel bad (we don't!) about deleting the explorer from your notebook when you're done with it, its not isn't to clutter your notebooks, it's to enable you to even more easily explore data and collect insights."
```
Comment (jbednar) [inline examples/tutorial/02_Plotting.ipynb]:
```suggestion
    "Don't feel bad (we don't!) about deleting the explorer from your notebook when you're done with it, it's not meant to clutter your notebooks, it's to enable you to even more easily explore data and collect insights."
```

File: examples/tutorial/06_Dashboards.ipynb [hunk 2]
```diff
@@ -385,11 +421,26 @@
...
+    "## Sharing\n",
+    "\n",
+    "Once you are happy with the dashboard you have built, it's time to share it! You can find on [Panel's website](https://panel.holoviz.org/how_to/index.html#share-your-work) more details about how to:\n",
+    "\n",
+    "- configure the server\n",
+    "- deploy the application on various platforms (e.g. Microsoft Azure, Google Cloud Platform, Hugging Face)\n",
+    "- save a dashboard as a static HTML file\n",
+    "- deploy a dashboard leveraging a new technology called WebAssembly thanks to which (and to Pyodide or PyScript) a Python application can run entirely in the browser, making it a lot easier to deploy a dashboard as you no longer need to bother setting up a server\n",
+    "\n",
+    "As it's pretty new and not documented yet on Panel's website, you can now easily deploy Panel dashboards on [Anaconda Notebooks](https://anaconda.cloud/anaconda-tools).\n",
```
Comment (jbednar) [inline examples/tutorial/06_Dashboards.ipynb]:
```suggestion
    "Though this feature is brand new as of 6/2023 and not yet documented on Panel's website, you can now easily deploy Panel dashboards on [Anaconda Notebooks](https://anaconda.cloud/anaconda-tools) as well.\n",
```

File: examples/tutorial/06_Dashboards.ipynb [hunk 3]
```diff
@@ -385,11 +421,26 @@
...
+    "\n",
+    "## Lumen\n",
+    "\n",
+    "Panel is a pretty free-form kind of library to build dashboards, it provides all the required building blocks that you then assemble as you like. The [Lumen library](https://lumen.holoviz.org/) is a relatively new addition to the HoloViz ecosystem that is much more opinionated about what a data app is. Building on top of [Intake](https://intake.readthedocs.io/en/latest/index.html) Panel, Lumen can take care of ingesting, filtering, transforming and viewing data, all through a declarative specification that can be stored in YAML files.\n",
```
Comment (jbednar) [inline examples/tutorial/06_Dashboards.ipynb]:
```suggestion
    "Panel is a highly flexible library to build apps and dashboards that provides all the required building blocks for you to assemble as you like. The [Lumen library](https://lumen.holoviz.org/) is a relatively new addition to the HoloViz ecosystem that is much more opinionated about what a data app is, handling many of the tasks involved automatically. Building on top of [Intake](https://intake.readthedocs.io/en/latest/index.html) Panel, Lumen can take care of ingesting, filtering, transforming and viewing data, all through a declarative specification that can be stored in YAML files.\n",
```

**Review by jbednar — APPROVED** [review]
Looks great, thanks! Can you strip those id tags out to make the actual changes more visible, and then please merge?

**Comment by maximlt** [comment]
> Can you strip those id tags

Unfortunately we will have to live with the `id` tags as future versions of the notebook format will require them. At some point we'll certainly make an update of all the notebooks across the repos to add these `id` tags, so that they don't pollute PRs like this one. That will require some tooling update (possibly in the `clean_update` pre-commit hook made by Simon). Once added the tags are no longer modified when you re-open a notebook, it'll be a one time cost.

**Pull request merged**